### PR TITLE
Feature/delete temp in clean script

### PR DESCRIPTION
**Pull request by natalyShvartzman**
Add parameter for remove all items in tmp folder older than X number of days

### CODE REVIEW

File: utils/cleaners.py [hunk 1]
```diff
@@ -27,7 +26,6 @@
     Database,
     Task,
     Sample,
-    TASK_RUNNING,
```
Comment (kfirstri) [inline utils/cleaners.py:30]: why was this deleted?

File: utils/cleaners.py [hunk 2]
```diff
@@ -358,6 +356,39 @@ def cuckoo_clean_lower_score(args):
     resolver_pool.map(lambda tid: delete_data(tid), id_arr)
 
 
+def tmp_clean_before_day(args):
+    """Clean up tmp folder
+    It deletes all items in tmp folder before now - days.
+    """
+    if not args.delete_tmp_items_older_than_days:
+        log.info("No days argument provided bailing")
```
Comment (kfirstri) [inline utils/cleaners.py]: I know it's the same like the other comments, but lets write a nicer info message

File: utils/cleaners.py [hunk 3]
```diff
@@ -358,6 +356,39 @@ def cuckoo_clean_lower_score(args):
...
+    for root, directories, files in os.walk(tmp_folder_path, topdown=True):
+        for name in files + directories:
+            path = os.path.join(root, name)
+            last_modified_time_in_seconds = os.stat(os.path.join(root, path)).st_mtime
+            file_time = today - datetime.fromtimestamp(last_modified_time_in_seconds)
+
+            if file_time.days > days:
+                try:
+                    if os.path.isdir(path):
+                        log.info("Delete file: {0}".format(path))
```
Comment (kfirstri) [inline utils/cleaners.py]: I think the info message is switched between the two conditions, this is the folder condition

**Review by kfirstri — APPROVED** [review]

File: utils/cleaners.py (hunk 1 above)
Comment (natalyShvartzman) [reply utils/cleaners.py:30]: Nobody use it
